model/rpc/spans.yaml

```diff
@@ -61,44 +61,34 @@ groups:
       - ref: network.type
         requirement_level: recommended
 
-  - id: span.rpc.grpc
-    type: span
-    stability: experimental
-    extends: rpc
+  - id: rpc.grpc.attributes
```

Comment:
`rpc.*flavor` spans capturing both client and server. If we want to preserve them as spans, we should break them down into two definitions. Otherwise they could stay as attribute groups.